src/pages/Integrations/Create/schema.ts

```diff
@@ -22,7 +22,7 @@ export const schema = (category, isEdit, isBehaviorGroupsEnabled) => ({
           IntegrationCategory.COMMUNICATIONS,
           IntegrationCategory.REPORTING,
         ].includes(category)
-          ? [integrationTypeStep(category, isEdit)]
+          ? [IntegrationTypeStep(category, isEdit)]
```

Comment:
You should send the acive feature flag in this call, similar to how `isBehaviorGroupsEnabled` is passed down.
```suggestion
          ? [IntegrationTypeStep(category, isEdit, { 'platform.integrations.pager-duty': isPagerDutyEnabled})]
```